www/api.js

```diff
@@ -1,143 +1,19 @@
-    'is_self': true,
-    'id': '60tlfl',
-    'gilded': 0,
-    'title': 'Planning on getting Captain Sonar to the table with the full 8-players. Would love some advice',
-    'url': 'https://www.reddit.com/r/boardgames/comments/60tlfl/planning_on_getting_captain_sonar_to_the_table/',
-    'author': '4fer6',
-    'created': 1490203502.0,
-    'score': 15,
-    'over_18': false
-  }
-]
-
-const comments = [
-  {
-    'author': 'hh8mr',
-    'body': 'comment 2',
-    'created': 1488274311.0,
-    'edited': false,
-    'gilded': 0,
-    'id': 'deaw2vi',
-    'link_id': '5th7sz',
-    'score': 1,
-    'subreddit_id': '3imv0'
-  },
-  {
-    'author': '74344',
-    'body': 'Your submission has been removed.\n\nViolation of Rules #2: \n\nNo rehosting allowed without explicit permission, unless it is obvious that the host allows it. Rehosting for the purposes of offering a direct link to an image is allowed in the comments.\n\n\n\nIf you feel that it has been removed in error, please [message us](http://www.reddit.com/message/compose?to=%2Fr%2Fprogrammerhumor) so that we may review it.',
-    'created': 1490217634.0,
-    'edited': false,
-    'gilded': 0,
-    'id': 'df9c4sc',
-    'link_id': '60on5c',
-    'score': 2,
-    'subreddit_id': '2tex6'
-  },
-  {
-    'author': '4fer6',
-    'body': 'Some random advice for running a good first game with 8 players.\n\n* As /u/raiden001 has stated, add the house rule that Captains must clearly state "Heading X" for every command. They must also wait for an OK from Engineer and First Mate before issuing another order.\n\n* Sit the players so the Captain is sat near or opposite the enemy Radio Operator. This will give the Radio Op a chance.\n\n* Double check all players are using the correct board and correct site of board. This seems obvious, but a little mistake like this will break the game and cause great frustration.\n\n* I would rate difficulty of each role in the following order (from most challenging to least challenging); Radio Op, Captain, Engineer, First Mate. Try to assign roles based on peoples comfort, and make sure the Radio Ops can hear the enemy Captain okay.\n\nCommonly missed rules;\n\n* You can\'t cross over your own path (like snake).\n\n* You can\'t use a system if any of the corresponding symbols are currently broken by engineer\n\n* You must check that you can use a system before calling stop\n',
-    'created': 1490218064.0,
-    'edited': false,
-    'gilded': 0,
-    'id': 'df9cfop',
-    'link_id': '60tlfl',
-    'score': 5,
-    'subreddit_id': '2qh55'
-  }
-]
+export function getUsers (callback, options) {
```

Comment:
It may be a good idea to use the [standard callback conventions](http://fredkschott.com/post/2014/03/understanding-error-first-callbacks-in-node-js/) like: the callback is always the last argument and the callback has two parameters: `error` and `result`. Alternatively since axis supports promises we could just return the promise from these functions instead of using callbacks.